.gitignore

```diff
@@ -0,0 +1,7 @@
+# CMake folders
+build
+build_*
+
+# OS generated files
+.DS_Store
+*.swp
```

Comment:
Newline at end of file

Reply:
I copied these files from ign tools they didn't have new lines, so I didn't add them.

Reply:
Added